src/panels/lovelace/components/hui-theme-select-editor.ts

```diff
@@ -17,6 +18,18 @@ export class HuiThemeSelectionEditor extends hassLocalizeLitMixin(LitElement) {
     };
   }
 
+  protected constructor() {
+    super();
+    this._initializing = true;
+  }
+  protected async update(changedProps) {
```

Comment:
I'm not sure what the parameter type should be, but the return type should be `Promise<void>`